### PR TITLE
Add Batteries v2.0.1

### PR DESCRIPTION
**Pull request by iplus26**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by miccal** [comment]
Please add a [`version`](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/version.md), [`sha256`](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md) and an [`appcast`](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Comment by iplus26** [comment]
> Please add a [`version`](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/version.md), [`sha256`](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md) and an [`appcast`](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

@miccal Thank you for references, [`atom.rb`](https://github.com/Homebrew/homebrew-cask/blob/645dbb8228ec2f1f217ed1431e188687aac13ca5/Casks/atom.rb#L7) in `appcast` document gives a really good example! 
I've updated the cask, pls check it out later when you have time

File: Casks/batteries.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  sha256 '9b4ec9875d6977392c182a9032ebead5857713bf9f78f7e78df29a7d8c592c1f'
+
+  # github.com/ronyfadel/BatteriesReleases was verified as official when first introduced to the cask
+  url "https://github.com/ronyfadel/BatteriesReleases/releases/download/v#{version}/Batteries.dmg"
+  appcast 'https://github.com/ronyfadel/BatteriesReleases/releases.atom'
+  name 'Batteries'
+  homepage 'https://www.fadel.io/batteries/'
+
+  app 'Batteries.app'
+end
```
Comment (ran-dall) [inline Casks/batteries.rb:12]:
```suggestion
  depends_on macos: '>= :mojave'

  app 'Batteries.app'
end

**Review by ran-dall — CHANGES_REQUESTED** [review]
@iplus26 This depends on `10.4` which is `Mojave`. 

Also, you may need to add an `zap` stanza.

**Review by ran-dall — APPROVED** [review]
@iplus26 Good work! Thank you for adding that!

**Comment by iplus26** [comment]
@ran-dall Thank you! It's night here so good night! 🌃

**Pull request merged**